examples/values-sigstore-openshift.yaml

```diff
@@ -45,4 +45,10 @@ scaffold:
       http:
         hosts:
           - host: tuf.$OPENSHIFT_APPS_SUBDOMAIN
-            path: /
\ No newline at end of file
+            path: /
+
+configs:
+  pull_secret:
+    secret_create: false
```

Comment:
with `secret_create` being set to false, the pull secret is never actually being created.  When I set it to be `true`, the following is created:

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: pull-secret
  namespace: sigstore-monitoring
type: Opaque
data:
  pull-secret.json: map[]
```

Reply:
this should be adressed from switching the data type to a string which has been done.